app/assets/stylesheets/components.scss

```
@@ -0,0 +1,19 @@
+// Place all the styles related to the components controller here.
+// They will automatically be included in application.css.
+// You can use Sass (SCSS) here: http://sass-lang.com/
+
+.app-certificate-tag-expiring .govuk-tag {
+  background-color: #CB4B0B;
+}
+.app-certificate-tag .govuk-tag {
+  float: right;
+}
+.side-navigation ul .active {
+  font-weight: bold;
+}
+.side-navigation ul li a {
+  text-decoration: none;
+}
+.app-table__body--noheading { 
+  border-top: 1px solid #bfc1c3 
```

we shouldn't be using hex codes, but the variables our CSS framework provides
```suggestion
  border-top: 1px solid $govuk-border-colour 
```